tests/integration/dhcp_test.py

```diff
@@ -148,6 +163,43 @@ def test_dhcp_with_addresses(dhcp_env):
     assertlib.assert_state(desired_state)
 
 
+def test_dhcp_for_bond_with_ip_address_and_slave(dhcp_env,
+                                                 setup_remove_bond99):
+
+    desired_state = {
+        INTERFACES: [
+            {
+                'name': 'bond99',
+                'type': 'bond',
+                'state': 'up',
+                'ipv4': {
+                    'enabled': True,
+                    'dhcp': True,
+                    'address': [
+                        {'ip': '192.168.122.250', 'prefix-length': 24}
```

Comment:
Please use rfc5737 addresses here and even better use a constant as you do in https://github.com/nmstate/nmstate/blob/master/tests/integration/static_ip_address_test.py#L26

Reply:
Done